### PR TITLE
Static page cleanup

### PR DESCRIPTION
**Pull request by samanthaandrews**
- Addresses a few content updates from Miguel
- Merges the new Header & Footer into the MDX pages

### CODE REVIEW

File: src/marketing/mdx/MDXComponents.tsx [hunk 1]
```diff
@@ -37,7 +30,21 @@ const H1 = ({ children }: MDXComponentProps) => (
 const H2 = ({ children }: MDXComponentProps) => {
   const { id } = createTOCAnchors(children);
   return (
-    <Text textStyle="h4" as="h2" marginTop="8" marginBottom="2" id={id}>
+    <Text
+      textStyle="h4"
```
Comment (Toxiapo) [inline src/marketing/mdx/MDXComponents.tsx:34]: the h4 style is much bigger than h2 here, this is due to the fact that I didn't specify the style for the mobile screens.

**Review by Toxiapo — CHANGES_REQUESTED** [review]
Just that one style update otherwise this is good to go!

File: src/marketing/mdx/MDXComponents.tsx (hunk 1 above)
Comment (samanthaandrews) [reply src/marketing/mdx/MDXComponents.tsx:34]: you're right. fixed!

**Review by Toxiapo — APPROVED** [review]

File: src/marketing/components/MarketingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import SiteHead from "components/Head";
+import { SWRConfig } from "swr";
+import swrConfig from "utils/swrConfig";
+import { UserProvider } from "components/context/UserContext";
+
+/* Wrapper for marketing pages that doesn't include references to theme-ui */
+
+const MarketingPage: React.FC = ({ children }) => {
```
Comment (kristojorg) [inline src/marketing/components/MarketingPage.tsx:9]: Why have this as separate from `MarketingPageLayout`?

File: src/marketing/components/MarketingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <>
+      <SiteHead />
+      <SWRConfig value={swrConfig}>
+        <UserProvider>{children}</UserProvider>
```
Comment (kristojorg) [inline src/marketing/components/MarketingPage.tsx:14]: I'm actually wondering if we should use the `<UserProvider>` here or the `<CredentialsProvider>`. We are only trying to determine the auth state of the user. The user provider has to fetch loans (takes forever) to do that, whereas we could also just assume that if the credentials are defined and set then the user is logged in... if they turn out to be wrong or something then the user will get redirected to login anyway once they try to go to the /app page

File: src/marketing/icon/BackToTop.tsx [hunk 4]
```diff
@@ -1,10 +1,8 @@
-import { SxProp } from "theme-ui";
 import React from "react";
+import { Icon } from "@chakra-ui/react";
 
-const BackToTopIcon = (props: React.SVGProps<SVGSVGElement> & SxProp) => (
-  <svg
-    width="16"
-    height="18"
+const BackToTop = (props: React.ComponentProps<typeof Icon>) => (
+  <Icon
```
Comment (kristojorg) [inline src/marketing/icon/BackToTop.tsx:5]: Oh this is good to know! I hadn't realized we could just use the `Icon` component this way

File: src/marketing/mdx/MDXComponents.tsx [hunk 5]
```diff
@@ -37,7 +30,21 @@ const H1 = ({ children }: MDXComponentProps) => (
...
+      marginBottom="2"
+      id={id}
+      _before={{
+        display: "block",
+        content: '" "',
+        marginTop: "-3em",
+        height: "3em",
+        visibility: "hidden",
+        pointerEvents: "none"
+      }}
```
Comment (kristojorg) [inline src/marketing/mdx/MDXComponents.tsx:46]: What is this for?

**Review by kristojorg — APPROVED** [review]
A couple questions, but I'm going to merge this and we can discuss on the base theme PR

**Pull request merged**